example/main.cpp

```diff
@@ -1,41 +1,48 @@
+#include <functional>
 #include <libdio/display.h>
 #include <libdio/input.h>
+#include <thread>
+#include <vector>
 
 using namespace std;
 
 int main() {
-    Display display(Display::Colors::PINK);
+    Display::init();
 
-    Display::showText("Test", Display::Colors::GREEN);
+    Display::DString display(Display::Color::PINK);
+    display << "Test\n";
+    display.print();
+
+    display.setColor(Display::Color::GREEN) << "In green\n";
+    display.print();
 
     Display::saveCursorPosition();
-    display.setContent("Hello World!\nNext line");
-    display.show();
+    display << "Hello World!\nNext line\n";
+    display.print();
+
+    this_thread::sleep_for(1000ms);
```

Comment:
> If you're already into changing the example/main.cpp, which is copied to the README, you could also remove the sleeps - I don't really think they're useful. And when I'm testing, it's just inconvenient to have to wait for 5 seconds :)

Reply:
Found a way to make it work on Linux. But not on MacOS CLion...